src/OpenTelemetry.Api/Trace/ActivityContextExtensions.cs

```diff
@@ -16,7 +16,7 @@
 
 using System.Diagnostics;
 
-namespace OpenTelemetry.Context.Propagation
+namespace OpenTelemetry.Trace
```

Comment:
`ActivityContext` does not expose `IsValid` as required by spec. So this extension method is compensating for that. Question is: What namespace should this use? 
I changed to OpenTelemetry.Trace as everything else required for Tracing/Activity is in that namespace. But the context/propagation classes who require this are in OpenTelemetry.Context.Propagation ns..

Reply:
In this case - `OpenTelemetry`.